dev/com.ibm.ws.jdbc_fat_oracle/fat/src/com/ibm/ws/jdbc/fat/oracle/OracleTraceTest.java

```diff
@@ -88,4 +90,12 @@ public void testConnectionPropsEmbeddedPasswordNotLogged() throws Exception {
         if (sslPasswords > 0)
             fail("SSL passwords logged in trace " + sslPasswords + " times. SSL Password: " + SSL_PASSWORD);
     }
+
+    @Test
+    public void testReadOnlyInfo() throws Exception {
+        server.setTraceMarkToEndOfDefaultTrace();
+        runTest(server, JEE_APP + "/" + SERVLET_NAME, getTestMethodSimpleName());
+        List<String> occurrences = server.findStringsInLogsAndTraceUsingMark("DSRA8207I");
+        assertEquals("The DSRA8207I message should have only been logged exactly once in log, and once in trace.", 2, occurrences.size());
```

Comment:
The test should check messages.log only (I think there is a similar findStringsInLogsUsingMark method for this) so that we are only testing the externals.  I'd rather not have tests that are vulnerable to changes in trace and it also costs more time to scan through traces.